controlplane/kubeadm/internal/machine_set.go

```diff
@@ -0,0 +1,246 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Modified copy of k8s.io/apimachinery/pkg/util/sets/int64.go
+// Modifications
```

Comment:
It might be worth deleting any unused functions on this type. My editor tells me HasAll, HasAny, Difference, Union, Intersection, Equal, and PopAny are all unused. With this modification section we can easily find the original source of those functions and add them if we need them in future.

Reply:
sgtm...  I'd really like to find a better name than MachineSet as well, but I'm drawing blanks there.